site/src/pages/docs/client-factory.mdx

```diff
@@ -34,6 +36,28 @@ You can also use <type://ClientFactoryOptions> to build <type://ClientFactory>.
 It's more verbose than calling a dedicated setter method, but it can be useful when you need to
 set a series of options programmatically.
 
+## Customizing TLS
+
+You can configure the TLS key and certificates
+for [mutual TLS (mTLS)](https://www.cloudflare.com/ko-kr/learning/access-management/what-is-mutual-tls/)
+using <type://ClientFactoryBuilder#tls(InputStream,InputStream)> .
+Additionally, you can specify various options such as a different CA certificate chain or enforce certain 
+cipher suites with <type://ClientFactoryBuilder#tlsCustomizer(Consumer)> .
```

Comment:
Maybe mention that this is for client authentication (for search engine indexing if nothing else)